aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -162,6 +163,7 @@ module aptos_framework::coin {
 
     /// Creates a new aggregatable coin with value overflowing on `limit`. Note that this function can
     /// only be called by Aptos Framework (0x1) account for now becuase of `create_aggregator`.
+    #[deprecated]
```

Comment:
Dumb question. Does adding the deprecated flag affect the functionality in anyway? Or does it just show a warning to the developer that this function is deprecated when he uses it?

Reply:
I think it's only a compiler warning

Reply:
Yes, just a compiler warning.